schemas/definitions.json

```diff
@@ -269,43 +272,52 @@
         "guidelines": [
           ["Must start with a capital letter."],
           ["Must end with a period."],
-          ["Should describe the data product, not the snapshot (i.e. the subset of data we extract from the data product)."]
+          ["Must not mention other metadata fields like `producer` or `version_producer`.", {"type": "exceptions", "value": ["Other metadata fields are crucial in the description of the data product."]}],
+          ["Should describe the data product, not the snapshot (i.e. the subset of data we extract from the data product)."],
+          ["Should ideally contain just one or a few paragraphs, that describe the data product succinctly."],
+          ["If the producer provides a good description, use that, either exactly or conveniently rephrased."]
         ],
         "category": "dataset"
       },
       "title_snapshot": {
         "title": "Title of the snapshot",
         "type": "string",
-        "description": "Title of the snapshot (i.e. the subset of data that we extract from the data product) which in most cases will coincide with the title of the data product.",
+        "description": "Title of the snapshot (i.e. the subset of data that we extract from the data product).",
         "examples": [
+          "Global Carbon Budget - Fossil fuels",
           "Neutron star mergers"
         ],
         "examples_bad": [
           [
+            "Global Carbon Budget",
             "Neutron star mergers (NASA, 2023)",
             "Data on neutron star mergers",
             "Neutron star mergers dataset"
           ]
         ],
```

Comment:
To correctly render the list of examples in the docs or wizard we should either:

- not have any `examples` or `examples_bad`
- just have `examples`
- have the same number of `examples` and `examples_bad`.

Here I'd do sth like


```json
"examples": [
  "Global Carbon Budget - Fossil fuels",
  "Neutron star mergers"
],
"examples_bad": [
  [
    "Global Carbon Budget",
  ],
  [
    "Neutron star mergers (NASA, 2023)",
    "Data on neutron star mergers",
    "Neutron star mergers dataset"
  ]
],
```


(see how it is done for `attribution_short`)

Reply:
Right, I may have other examples that don't fulfil this rule.